src/components/settings-table/dataset-table-row.js

```diff
@@ -1,46 +1,39 @@
 import React from 'react'
 
-import TableRow from '@material-ui/core/TableRow'
-import TableCell from '@material-ui/core/TableCell'
-import InputNumber from '../input-number'
 import i18n from '@dhis2/d2-i18n'
-import dataTableStyles from '../../styles/DataTable.module.css'
 import tableTitleStyles from '../../styles/TableTitle.module.css'
+import { InputNumber } from '../inputs'
+import TableRow from './table-row'
+import { Divider } from '@dhis2/ui'
 
 const DataSetTableRow = ({
     dataRow,
     periodType,
     defaultValue,
     states,
     onChange,
-}) => {
-    return (
+}) => (
+    <div>
```

Comment:
It's better to use `<>` or `<React.Fragment>` here, instead of rendering an extra Div unnecessarily